javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var click = 0;
+var $bulbOn = document.querySelector('.bulb-on');
+var $backgroundOn = document.querySelector('.background-on');
+
+function bulbClick(event) {
+  click++;
+  if (click % 2 === 0) {
```

Comment:
To determine whether to turn the light on or off, you are making a counter, but the only thing you ever care about is if the counter's value was even or odd. This is a situation where you would be better off using a boolean to keep track of if the light is currently on.